mls_validation_service/src/handlers.rs

```diff
@@ -81,6 +83,25 @@ impl ValidationApi for ValidationService {
             responses: out,
         }))
     }
+
+    async fn get_association_state(
+        &self,
+        request: Request<GetAssociationStateRequest>,
+    ) -> Result<Response<GetAssociationStateResponse>, Status> {
+        let updates = request
+            .into_inner()
+            .updates
+            .into_iter()
+            .map(IdentityUpdate::from_proto)
+            .collect::<Result<Vec<_>, _>>()
+            .map_err(|e| Status::new(Code::Cancelled, e.to_string()))?;
```

Comment:
Canceled doesn't feel like the right GRPC status code for this error. What about `Code:: InvalidArgument`